uvicorn/protocols/http/h11_impl.py

```diff
@@ -497,10 +489,10 @@ async def send(self, message: "ASGISendEvent") -> None:
 
             # Write response status line and headers
             reason = STATUS_PHRASES[status_code]
-            event = h11.Response(
-                status_code=status_code, headers=headers, reason=reason
-            )
-            output = self.conn.send(event)
+            # fmt: off
+            response = h11.Response(status_code=status_code, headers=headers, reason=reason) # noqa: E501
```

Comment:
I'm starting to have the feeling that 88 is far worse than 120 for `uvicorn`.

Reply:
These fmt on/off E501 ignores are definitely sad to see. Why not split the response construction over multiple lines? Maybe you _should_ bump the line length for the project, although 120 chars is pretty long.

Reply:
Yeah, I'm going to remove them.